### PR TITLE
refact(event): refactor event manager

### LINKED ISSUE #302: Design and implement the scheduler for event
Based on the investigation of #286, design and implement the scheduler for event. 

**Tasks:**
- [x] Implement the event manager based on MongoDB instead of ETCD #324 
- [x] Clean up the config, doc and Shell script about ETCD #325

### PR DESCRIPTION
**Pull request by supereagle**
## Fixes #302

**Changes**

  - [x] refactor event manager to remove ETCD

### CODE REVIEW

File: store/event.go [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+	// retryInterval represents the interval unit for retry.
+	retryInterval = 1 * time.Minute
+)
+
+var (
+	// outTimeout represents the timeout for event to keep "out" status.
+	// After the event is out, then its status will be "handling" after it starts to be handled,
+	// otherwise, it will be re-enqueued and its status will change back to "in".
+	outTimeout, _ = time.ParseDuration("-3m")
```
[the remark this answers is not in the document]
Comment (zoumo) [reply store/event.go]: Is there any reason to use time.ParseDuration? It seems that you exactly know the timeout duration

File: store/event.go [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+	event.QueueStatus = api.InQueue
+
+	var intervalNum int
+	if event.Retry <= retryThreshold {
+		intervalNum = event.Retry
+	} else {
+		intervalNum = retryThreshold
+	}
+
+	event.InTime = event.InTime.Add(time.Duration(time.Duration(intervalNum) * retryInterval))
```
Comment (zoumo) [inline store/event.go]: event.InTime = event.InTime.Add(intervalNum * retryInterval)

File: store/event.go (hunk 1 above)
Comment (zoumo) [inline store/event.go]: Is there any reason for you to use time.ParseDuration? It seems that you exactly know the timeout duration.

Comment (supereagle) [reply store/event.go]: Fixed

**Pull request merged**